lib/publish.sh

```diff
@@ -26,6 +26,7 @@
 
 publish_release() {
 	mv ${HBSD_STAGEDIR} ${HBSD_PUBDIR}/${HBSD_BUILDNUMBER}
```

Comment:
Make sure to check for the exit code (> 0 being error) for `mv` here.

Reply:
how does this look? rsync of the parent dir takes care of the the symlink and all the builds